static-site/data/BaseConfig.js

```diff
@@ -0,0 +1,31 @@
+// This file contains basic configuration information that is used in
+// non-React contexts, such that putting it into `SiteConfig.jsx`
+// would not work. Everything that is exported from this file will
+// be re-exported from `SiteConfig.jsx`.
+
+export const siteTitle = "Nextstrain";
+
+// Alternative site title for SEO.
+export const siteTitleAlt = "Real-time tracking of pathogen evolution";
+
+// Logo used for SEO
+export const siteLogo = "/nextstrain-logo-small.png";
+
+export const siteUrl = "https://nextstrain.org";
+
+// Website description used for RSS feeds/meta description tag.
+export const siteDescription = "Real-time tracking of pathogen evolution";
+
+export const groupsApp = false;
+
+export const groupsTitle = "Scalable Sharing with Nextstrain Groups";
+
```

Comment:
I believe CDC's internal "groups app instance" has customized these in SiteConfig.jsx. This would be a breaking change for them, which should be fine as long as we give them a heads up on the `#nextstrain-cdc` Slack channel.

Reply:
Tracked in #1001